controllers/users.js

```diff
@@ -331,6 +331,14 @@ const rejectProfileDiff = async (req, res) => {
 const addUserIntro = async (req, res) => {
   try {
     const rawData = req.body;
+    const joinData = await userQuery.getJoinData(req.userData.id);
+
+    if (joinData.length === 1) {
```

Comment:
what happens if a person added 3 join data before your PR goes in?
wont the length === 1 fail?

Reply:
> what happens if a person added join data thrice before your PR goes in? won't the length === 1 fail?

If the user has previous records it will fetch one result which will be length === 1 and returns from there on.